packages/sommelier.rb

```diff
@@ -3,92 +3,93 @@
 
-  binary_url ({
-     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-20210109-chromeos-armv7l.tar.xz',
-      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-20210109-chromeos-armv7l.tar.xz',
-        i686: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-20210109-chromeos-i686.tar.xz',
-      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-20210109-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-20210109-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-20210109-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-20210109-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/sommelier-20210109-1-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-     aarch64: 'f7c8a2ceed9ba558b847b74648e0cf3b9911f52f535131ed4f4c6537158986f8',
-      armv7l: 'f7c8a2ceed9ba558b847b74648e0cf3b9911f52f535131ed4f4c6537158986f8',
-        i686: 'e280ae66333a01f6406333aa6a7836b6d2638ca25aeb81378f55e1bd9df9f339',
-      x86_64: '5348afbfceeed2ddf5078d4d325f50ad736ea89c8249db263009c3d23cd05db9',
+  binary_sha256({
+    aarch64: 'c076c1f99a018eb4c75de3768445e322bcfbc2221942b967a928924ec798e6b0',
+     armv7l: 'c076c1f99a018eb4c75de3768445e322bcfbc2221942b967a928924ec798e6b0',
+       i686: '98d5f94ef0f633da694b1525d1e01ad8ddf82fbb14996c52eec6b7ef51f5fed7',
+     x86_64: '45cf6f34a6fc6bd1bb5238eba9fb7dfbf5e528c6496dcad2d9048aeec4ac20ed'
   })
 
+  depends_on 'coreutils' # for readlink in wrapper script
+  depends_on 'libdrm'
+  depends_on 'libxcb'
+  depends_on 'libxcomposite' => :build
+  depends_on 'libxfixes' => :build
+  depends_on 'libxkbcommon'
+  depends_on 'llvm' => :build
   depends_on 'mesa'
-  depends_on 'xkbcomp'
-  depends_on 'xorg_server' unless File.exists? "#{CREW_PREFIX}/bin/Xwayland.elf"
-  depends_on 'xwayland'
+  depends_on 'pixman'
+  depends_on 'procps' # for pgrep in wrapper script
   depends_on 'psmisc'
-  depends_on 'vim' # Until there is a standalone package providing xxd
-  depends_on 'xdpyinfo'
-  depends_on 'xauth'
-  depends_on 'xsetroot'
-  depends_on 'llvm' => :build
+  depends_on 'wayland'
+  depends_on 'xdpyinfo' # for xdpyinfo in wrapper script
+  depends_on 'xsetroot' # for xsetroot in sommelierrc script
+  depends_on 'xwayland'
+  depends_on 'xxd_standalone' # for xxd in wrapper script
 
   case ARCH
-   when 'armv7l', 'aarch64'
-      PEER_CMD_PREFIX='/lib/ld-linux-armhf.so.3'
-   when 'i686'
-      PEER_CMD_PREFIX='/lib/ld-linux-i686.so.2'
-   when 'x86_64'
-      PEER_CMD_PREFIX='/lib64/ld-linux-x86-64.so.2'
+  when 'armv7l', 'aarch64'
+    PEER_CMD_PREFIX = '/lib/ld-linux-armhf.so.3'.freeze
+  when 'i686'
+    PEER_CMD_PREFIX = '/lib/ld-linux-i686.so.2'.freeze
+  when 'x86_64'
+    PEER_CMD_PREFIX = '/lib64/ld-linux-x86-64.so.2'.freeze
```

Comment:
Do we have to use constants?  Wouldn't an `@` variable remove the `.freeze` restraint?

Reply:
I don't know if one is better than the other, but this should work fine w/ the @ variables... Let me change that.